build/integration-test.sh

```diff
@@ -49,7 +51,7 @@ Usage: ${0} <app-type>
 Where app-type is one of [short|all]:
   short: Runs e2e integration tests for part of apps
   all: Runs e2e integration tests for all apps
-  oc: Runs e2e integration tests for apps that are to be tetsed against openshift cluster
+  openshift ocp_version=<ocp_version>: Runs e2e integration tests for apps that are to be tetsed against openshift cluster, OCP version can be provided using ocp_version argument
```

Comment:
Should also add list of supported versions or valid values for `openshift ocp_version`